src/main/java/com/es/phoneshop/model/product/SortField.java

```diff
@@ -0,0 +1,6 @@
+package com.es.phoneshop.model.product;
+
+public enum SortField {
+    price,
```

Comment:
Enum value names should be the same as for constants, e.g. VALUE, MULTIWORD_VALUE, etc.